gtfs-realtime/CHANGES.md

```diff
@@ -17,29 +17,34 @@ When a producer or consumer is interested in adding a new field to the GTFS Real
 
 ### Specification amendment process
 1. Create a git branch with update of all relevant parts of protocol definition, specification and documentation files (except for translations).
-1. Create pull request on https://github.com/google/transit. Pull request must contain an extended description of the patch. The creator of the pull request becomes the _advocate_.
-1. Once pull request is registered, it must be announced by its advocate in the [GTFS Realtime mailing list](https://groups.google.com/forum/#!forum/gtfs-realtime). Once announced, the pull request is considered a proposal.
+2. Create pull request on https://github.com/google/transit. Pull request must contain an extended description of the patch. The creator of the pull request becomes the _advocate_.
+3. Once pull request is registered, it must be announced by its advocate in the [GTFS Realtime mailing list](https://groups.google.com/forum/#!forum/gtfs-realtime). Once announced, the pull request is considered a proposal.
   - Since the advocate is a contributor, they must sign the [Contributor License Agreement](../CONTRIBUTING.md) before pull request can be accepted.
-1. The discussion of the proposal follows. Pull request comments should be used as the sole discussion forum.
+4. The discussion of the proposal follows. Pull request comments should be used as the sole discussion forum.
   - The discussion lasts for as long as the advocate feels necessary, but must be at least 7 calendar days.
   - The advocate is responsible for timely update of the proposal (i.e. pull request) based on the comments for which they agree to.
   - At any point in time the advocate can claim proposal abandoned.
-1. The advocate can call for a vote on a version of the proposal at any point in time following the initial 7-day interval required for discussion.
-1. Vote lasts the minimum period sufficient to cover 7 full calendar days and 5 full Swiss business days. Vote ends at 23:59:59 UTC.
+5. The advocate can call for a vote on a version of the proposal at any point in time following the initial 7-day interval required for discussion.
+6. Vote lasts the minimum period sufficient to cover 7 full calendar days and 5 full Swiss business days. Vote ends at 23:59:59 UTC.
   - The advocate should announce the specific end time at the start of the vote.
   - During voting period only editorial changes to the proposal are allowed (typos, wording may change as long as it does not change the meaning).
   - Anyone is allowed to vote yes/no in a form of comment to the pull request, and votes can be changed until the end of the voting period.
     If a voter changes her vote, it is recommended to do it by updating the original vote comment by striking through the vote and writing the new vote.
+  - Comments containing a yes/no vote must also include the following information:
```

Comment:
I think we should push for GitHub user accounts to have a name and organization (if applicable) in the profile itself.  As @giocorti mentioned earlier this would also allow for easier analytics.  We could include some language like "If a user does not want to populate this information in their official GitHub profile, they may instead add it to their vote comment."

Reply:
Sounds good to me.